src/LanguageServer/Protocol/Handler/References/FindUsagesLSPContext.cs

```diff
@@ -138,15 +138,15 @@ await foreach (var reference in references)
                     // Each reference should be associated with a definition. If this somehow isn't the
                     // case, we bail out early.
                     if (!_definitionToId.TryGetValue(reference.Definition, out var definitionId))
-                        return;
+                        continue;
```

Comment:
>continue [](http://example.com/codeflow?start=24&length=8)

should the comment be updated?

Reply:
enh

we're still 'bailing out' on that reference.